packages/audio-block/src/AudioBlock.tsx

```diff
@@ -186,32 +190,38 @@ export const AudioBlock = ({ appBridge }: BlockProps) => {
             )}
             <div className="tw-flex tw-gap-4 tw-justify-between tw-w-full">
                 <div className="tw-flex-1">
-                    <div data-test-id="audio-block-title">
-                        <RichTextEditor
-                            designTokens={designTokens}
-                            border={false}
-                            onBlur={saveTitle}
-                            placeholder={isEditing ? 'Asset name' : undefined}
-                            readonly={!isEditing}
-                            value={title ?? DEFAULT_CONTENT_TITLE}
-                            plugins={customTitlePlugins}
-                        />
-                    </div>
-                    <div data-test-id="audio-block-description">
-                        <RichTextEditor
-                            designTokens={designTokens}
-                            border={false}
-                            position={Position.FLOATING}
-                            onBlur={saveDescription}
-                            placeholder={isEditing ? 'Add a description here' : undefined}
-                            readonly={!isEditing}
-                            value={description ?? DEFAULT_CONTENT_DESCRIPTION}
-                        />
-                    </div>
+                    {title !== DEFAULT_CONTENT_TITLE || isEditing ? (
+                        <div data-test-id="audio-block-title">
+                            <RichTextEditor
+                                designTokens={designTokens}
+                                border={false}
+                                onBlur={saveTitle}
+                                placeholder={isEditing ? 'Asset name' : undefined}
+                                readonly={!isEditing}
+                                value={title ?? DEFAULT_CONTENT_TITLE}
+                                plugins={customTitlePlugins}
+                                updateValueOnChange={true}
+                            />
+                        </div>
+                    ) : null}
+
+                    {description !== DEFAULT_CONTENT_DESCRIPTION || isEditing ? (
+                        <div data-test-id="audio-block-description">
+                            <RichTextEditor
+                                designTokens={designTokens}
+                                border={false}
+                                position={Position.FLOATING}
+                                onBlur={saveDescription}
+                                placeholder={isEditing ? 'Add a description here' : undefined}
+                                readonly={!isEditing}
+                                value={description ?? DEFAULT_CONTENT_DESCRIPTION}
```

Comment:
the type on line 44 is the wrong one. It is just `p` not `paragraph` => `const DEFAULT_CONTENT_DESCRIPTION = '[{"type":"p","children":[{"text":""}]}]';`

Reply:
oops, where did i get that from, adjusted